### PR TITLE
Add Azure Table storage to the list of store providers

### PR DESCRIPTION
**Pull request by aloneguid**
# Description

Extend documentation to add Azure Table Storage state store.

## Issue reference

#320 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

File: howto/setup-state-store/setup-azure-tablestorage.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Setup Azure Table Storage 
+
+## Creating an Azure CosmosDB account
```
Comment (yaron2) [inline howto/setup-state-store/setup-azure-tablestorage.md]: Remove CosmosDB reference here

File: howto/setup-state-store/setup-azure-tablestorage.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      key: <KUBERNETES-SECRET-KEY>
+  - name: tableName
+    value: <REPLACE-WITH-TABLE-NAME>
+```
+
+## Apply the configuration
+
+### In Kubernetes
+
+To apply the CosmosDB state store to Kubernetes, use the `kubectl` CLI:
```
Comment (yaron2) [inline howto/setup-state-store/setup-azure-tablestorage.md]: Change CosmosDB to Azure Table Storage

File: howto/setup-state-store/setup-azure-tablestorage.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+```
+kubectl apply -f cosmos.yaml
```
Comment (yaron2) [inline howto/setup-state-store/setup-azure-tablestorage.md]: Change cosmos to azuretable.yaml

File: howto/setup-state-store/setup-azure-tablestorage.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+### Running locally
+
+The Dapr CLI will automatically create a directory named `components` in your current working directory with a Redis component.
+To use CosmosDB, replace the redis.yaml file with azuretable.yaml file above.
```
Comment (yaron2) [inline howto/setup-state-store/setup-azure-tablestorage.md]: Change CosmosDB to Azure Table Storage

**Comment by aloneguid** [comment]
@yaron2 thanks, apologies, find&replace bug.

**Pull request merged**